python/models.md

```diff
@@ -0,0 +1,9 @@
+# Model style
+
+```
+title = models.CharField(
+    max_length=100,
+    blank=True,
+    null=True,
```

Comment:
@dan-gamble I believe the dangling comma is not used in JS-land because 1) historical reasons around IE (it would create an extra element at the end) and 2) because JSON strictly forbids the dangling comma.